base/applications/rapps/lang/ro-RO.rc

```diff
@@ -74,9 +75,9 @@ BEGIN
     EDITTEXT IDC_DOWNLOAD_DIR_EDIT, 15, 86, 166, 12, WS_CHILD | WS_VISIBLE | WS_GROUP | ES_AUTOHSCROLL
     PUSHBUTTON "Spe&cificare…", IDC_CHOOSE, 187, 85, 50, 14
     AUTOCHECKBOX "Șt&erge aplicația de instalare după instalarea programului", IDC_DEL_AFTER_INSTALL, 16, 100, 218, 12
-    GROUPBOX "Software source", -1, 4, 118, 240, 46
-    CONTROL "Use default", IDC_SOURCE_DEFAULT, "Button", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 15, 132, 74, 10
-    CONTROL "Specified source", IDC_USE_SOURCE, "Button", BS_AUTORADIOBUTTON | WS_TABSTOP, 15, 147, 74, 10
+    GROUPBOX "Sursa logicii computaționale", -1, 4, 118, 240, 46
```

Comment:
The average Romanian computer nerd or anybody else knows what a "software" is and we've been getting along with this word for quite such a long time. This literal translation just looks so odd and it's even misleading, `logicii computaționale`??

Reply:
I looked in the 15th rule from Romanian translation notes. But I can change it if you want.

Reply:
Or maybe, should I change it with the word "programul"? It's shorter than "logică computațională". The word "logică computațională" doesn't fit very well in the dialog box.

Reply:
> Or maybe, should I change it with the word "programul"? It's shorter than "logică computațională". The word "logică computațională" doesn't fit very well in the dialog box.

This ("programul") may foot the bill in this instance. Other than that, for both @GeoB99 and @AndreiMiloiu, I want to bring to your attention the fact that this is an operating system translation that is being done here. At some point, when it will get used widely, **its conventions will also be taken as reference** by others translating applications that run on it. It is important to be aware that your choices may have far reaching effects, and thus to take care of the quality of this potential reference base. If I would have translated a mere application, then I would have followed whatever flavor of consecrated tech glossary was being favored around, but here I could do (and did) more than that. And more than translating is what you (should assume that) are nowadays doing too. This is why it is important to have written rules and conventions for ReactOS. Back then I've chosen "logică computațională" by following ...well, logic. My bet was that once ReactOS use gets traction, it won't mater that whatever untranslated terms got in use or not until then. Whatever use of untranslated terms we currently have is the consequence of nobody caring enough. (BTW, the fact that you chose to translate is proof that you do care.) If, after all, you decide that my choices need change, please do not forget to update the translation notes and update the rest of ReactOS translation body as well.